README.md

```diff
@@ -17,24 +17,13 @@
     - [Overview](#Overview)
     - [Responsibilities of Contributor](#Responsibilities-of-Contributor)
     - [Responsibilities of Maintainer](#Responsibilities-of-Maintainer)
-    - [Sanity Checks after Building Release](#Sanity-Checks-after-Building-Release)
+    - [Safety Checks after Building Release](#Sanity-Checks-after-Building-Release)
```

Comment:
I am sorry for not doing this updated in the developer doc PR. Can you please change the path from `#Sanity-Checks-after-Building-Release` to `#Safety-Checks-after-Building-Release` so the link works? Thank you.

Reply:
I think I addressed this issue (and the other one you mentioned) with my latest commit. Can you check it again just to make sure, please?

Reply:
Thank you. It looks great!